build/templates/attributes.py.mako

```diff
@@ -59,16 +59,20 @@ class AttributeViBoolean(Attribute):
         session._set_attribute_vi_boolean(channel, self._attribute_id, value)
 
 
-class AttributeEnum(AttributeViInt32):
+class AttributeEnum(object):
 
-    def __init__(self, attribute_id, enum_meta_class, channel=''):
+    def __init__(self, underlying_attr_type, enum_meta_class, attribute_id, default_channel=''):
```

Comment:
On the same parameter list client passes two meta-classes: one for the Attribute and one for the End.
Can we make the name consistent? Either underlying_attr_meta_class or enum_type

Reply:
Changed to `underlying_attribute_meta_class`